### DISCUSSION TITLE
V23桌面右键菜单项设计冗余

### DISCUSSION THREAD

**Discussion by ankium** [linuxdeepin/developer-center#4483] [General | 常规讨论]
![avatar](https://storage.deepin.org/thread/202305232120124751_%E5%BE%AE%E4%BF%A1%E6%88%AA%E5%9B%BE_20230523211947.png)
如图，V23 Beta版本中鼠标右键“桌面选项”下二级菜单已经包含了“开启桌面整理”、“自动排列图标”和“图标大小”，但是在右键一级菜单中仍然存在“自动排列”、图标大小和“开启桌面整理”三项，这样的设计明显冗余！即便是为提高用户的操作效率，那也不必在如此近距离的范围内提供重复选项！其实只留“桌面选项”可能更合理，毕竟其中的三个选项并非高频操作！希望采纳并改进！

**Comment by BLumia** [comment]
cc @meiyixiang
